src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java

```diff
@@ -40,11 +41,19 @@ private void processFixture(Fixture fixture) {
     }
 
     private void handleFixtureInstructions(Fixture fixture) {
-        List<Instruction> fixtureInstructions = new ArrayList<>();
         val table = context.getConfig().table(fixture.name);
+        List<Instruction> fixtureInstructions = new ArrayList<>();
```

Comment:
Why do we need `fixtureInstructions` given we can put instructions into `context.getInstructions()` ?